quarkus/embedded/runtime/pom.xml

```diff
@@ -44,6 +44,12 @@
         <dependency>
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-core-graalvm</artifactId>
+            <exclusions>
```

Comment:
I think we should probably be excluding this from the `infinispan-core-graavlm` module so that non-quarkus users also aren't affected by the issue this addresses?

Reply:
this exclusion is related only to the native compilation in this module, you mean adding the exclusion at the infinispan-core-graalvm level ?

Reply:
I checked, I think this is something that remained by another change I did. just checking